pages/contests/index.js

```diff
@@ -0,0 +1,68 @@
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../../styles/Home.module.css";
+import { useState } from "react";
+
+function RenderContest({ contest }) {
+    return (
+        <>
+            <div class="max-w-sm rounded overflow-hidden shadow-lg">
+                <img class="w-full" src="https://www.jquery-az.com/html/images/banana.jpg" alt="" />
+                <div class="px-6 py-4 text-center">
+                    <div class="font-bold text-xl mb-2">{contest.name}</div>
+                    <p class="text-gray-700 text-base">
+                        {contest.description}
+                    </p>
+                    <p class="text-gray-700 text-base">
+                        <a>End date: </a>
+                        {contest.date_end.slice(0, 10)}
+                    </p>
+                </div>
+            </div>
+        </>
+    )
+}
+
+
+export default function Home({ contestList }) {
```

Comment:
same thing here, we don't need to always do styles.<something> if we deconstruct first

Reply:
done